### DISCUSSION TITLE
Placing functions in certain memory regions, and inlining

### DISCUSSION THREAD

**Discussion by mcbridejc** [modm-io/modm#826] [Ideas]
While working on [this PR](https://github.com/modm-io/modm/pull/824) I came across an issue where I declared a function with `modm_ramcode`, but then it got inlined into flash. I fixed it by adding the `noinline` attribute, but I wonder if this (noinline) should be the default behavior for `modm_ramcode`?

**Comment by salkinium** [comment]
Yes, it should, please fix it!

**Reply by salkinium** [reply]
Well, it should be added to [modm_fastcode](https://github.com/modm-io/modm/blob/develop/src/modm/architecture/utils.hpp#L158-L159) since that was aliased some time ago.
